pkg/sdn/plugin/controller.go

```diff
@@ -339,7 +340,12 @@ func (plugin *OsdnNode) SetupSDN(localSubnetCIDR, clusterNetworkCIDR, servicesNe
 func (plugin *OsdnNode) SetupEgressNetworkPolicy() error {
 	policies, err := plugin.registry.GetEgressNetworkPolicies()
 	if err != nil {
-		return fmt.Errorf("Could not get EgressNetworkPolicies: %s", err)
+		if kapierrs.IsForbidden(err) {
```

Comment:
can you also add a comment here explaining what server version this is maintaining compatibility for, and open a follow-up techdebt issue to track removing this in a future release?